data/pathfinder/paizo/player_companion/champions_of_purity/cop_abilities_class.lst

```diff
@@ -8,7 +8,7 @@ SOURCELONG:Champions of Purity	SOURCESHORT:COP	SOURCEWEB:http://paizo.com/produc
 # Ability Name		Unique Key						Category of Ability		Type								Description																																																																																																																																																											Bonus to Situation											Class Skill		Source Page		Aspects
 Aura of Sanctification	KEY:Domain Power ~ Aura of Sanctification	CATEGORY:Special Ability	TYPE:SpecialQuality.Supernatural.DomainPower	DESC:At 8th level, as an immediate action, you can emit a 30-foot-radius aura of sanctification for %1 rounds per day. These rounds do not need to be consecutive. Within this aura, effects that are specifically harmful to good-aligned creatures are inverted so that they no longer harm good creatures and instead harm evil creatures. Similarly, effects that are specifically beneficial to evil creatures instead become beneficial only to good creatures.|DomainGoodLVL																																																																																																SOURCEPAGE:p.21	ASPECT:CheckCount|%1|DomainGoodLVL	ASPECT:CheckType|Rounds per Day
 # Revelation Subdomain
-Guided Eyes			KEY:Domain Power ~ Guided Eyes		CATEGORY:Special Ability	TYPE:SpecialQuality.Supernatural.DomainPower	DESC:Perception is always a class skill for you. In addition, whenever you make a skill check to see through a disguise or find something that is hidden or concealed, you gain a +4 sacred bonus on the check.																																																																																																																										BONUS:SITUATION|See Through Disguise|4	BONUS:SITUATION|Find Hidden Object|4	CSKILL:Perception	SOURCEPAGE:p.21
+Guided Eyes			KEY:Domain Power ~ Guided Eyes		CATEGORY:Special Ability	TYPE:SpecialQuality.Supernatural.DomainPower	DESC:Perception is always a class skill for you. In addition, whenever you make a skill check to see through a disguise or find something that is hidden or concealed, you gain a +4 sacred bonus on the check.																																																																																																																										BONUS:SITUATION|See Through Disguise|4	BONUS:SITUATION|Notice Hidden Object|4	CSKILL:Perception	SOURCEPAGE:p.21
```

Comment:
Neither of these are correct for syntax. BONUS:SITUATION|SkillName=SituationName,SkillName=SituationName|Value|TYPE|PREXXX